docs/sources/enterprise/access-control-refactor/rbac-fixed-role-definitions.md

```diff
@@ -54,14 +56,3 @@ The following table lists permissions associated with each fixed role.
 | `fixed:annotations:reader`             | `annotations:read`                                                                                                                                                                                                                                                       | Read all annotations and annotation tags.                                                                                                                                                                                                                                             |
 | `fixed:annotations.dashboard:writer`   | `annotations:write` <br>`annotations.create`<br> `annotations:delete` for scope `annotations:type:dashboard`                                                                                                                                                             | Create, update and delete dashboard annotations and annotation tags.                                                                                                                                                                                                                  |
 | `fixed:annotations:writer`             | `annotations:write` <br>`annotations.create`<br> `annotations:delete` for scope `annotations:type:*`                                                                                                                                                                     | Create, update and delete all annotations and annotation tags.                                                                                                                                                                                                                        |
-
-## Built-in role assignments
```

Comment:
Did this move somewhere else, or is it being removed?

Reply:
I moved that content and neglected to remove the heading. All done now, thanks!